publish/publish.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
Must it be run from the root dir?

Reply:
I was running `./publish/publish.sh` but I could add an if check and cd if necessary.

Reply:
That's fine, I just wanted [this](https://github.com/forcedotcom/SalesforceMobileSDK-Package/pull/239) to be correct